Cargo.toml

```diff
@@ -1,29 +1,44 @@
 [package]
-name = "rodio"
-version = "0.14.0"
 authors = ["Pierre Krieger <pierre.krieger1708@gmail.com>"]
-license = "MIT OR Apache-2.0"
 description = "Audio playback library"
-keywords = ["audio", "playback", "gamedev"]
-repository = "https://github.com/RustAudio/rodio"
 documentation = "http://docs.rs/rodio"
 edition = "2018"
+keywords = ["audio", "playback", "gamedev"]
+license = "MIT OR Apache-2.0"
+name = "rodio"
+repository = "https://github.com/RustAudio/rodio"
+version = "0.14.0"
 
 [dependencies]
+claxon = {version = "0.4.2", optional = true}
 cpal = "0.13"
-claxon = { version = "0.4.2", optional = true }
-hound = { version = "3.3.1", optional = true }
-lewton = { version = "0.10", optional = true }
-minimp3 = { version = "0.5.0", optional = true }
+hound = {version = "3.3.1", optional = true}
+lewton = {version = "0.10", optional = true}
+minimp3 = {version = "0.5.0", optional = true}
+symphonia = {version = "0.3", optional = true, default-features = false}
+thiserror = "1.0.25"
```

Comment:
I'm not in favour of adding thiserror, it pulls in a huge set of dependencies for very little improvement in comfort. Plus it has nothing to do with symphonia addition so it should be done in a separate PR if at all.

Reply:
I added it because I needed to map the errors that Symphonia returns to a DecoderError and it made it a little easier but I can do that manually instead.